client/state/experiments/schema.js

```diff
@@ -0,0 +1,23 @@
+export const schema = {
+	type: 'object',
+	additionalProperties: true,
+	properties: {
```

Comment:
I've tested the following schemas and looks like it works.

```
isLoading: {
    type: 'boolean',
},
```
...
```
variations: {
    type: 'object',
    additionalProperties: false,
    patternProperties: {
        '^[a-z0-9_]+$': {
            oneOf: [
                  { type: 'string' },
                  { type: 'null' },
            ],
        },
    },
},
```

Reply:
Oh nice, thanks!